README.md

```diff
@@ -640,7 +640,12 @@ abstract syntax tree from formatting.
 For example:
 
 ```js
-matrix(1, 0, 0, 0, 1, 0, 0, 0, 1);
```

Comment:
The ignore comment needs to be above the fence otherwise it will show up in the rendered version.